OneSignalSDK/onesignal/core/src/main/java/com/onesignal/core/internal/http/impl/HttpClient.kt

```diff
@@ -110,7 +106,7 @@ internal class HttpClient(
         method: String?,
         jsonBody: JSONObject?,
         timeout: Int,
-        cacheKey: String?,
+        headerValues: OptionalHeaderValues?,
```

Comment:
`headerValues`, like noted below with the class name, the wording "value" doesn't add anything.